Max.java

```diff
@@ -5,9 +5,19 @@ static int max(int a, int b){
 		int max=a>b?(a):(b);
 		return max;
 	}
+// 	static int max(int a, int b){
+// 		int maximum;
+// 		if(a>b){
+// 			maximum=a;
+// 		}
+// 		else{
+// 			maximum=b;
+// 		}
+// 		return maximum;
+// 	}

```

Comment:
An another way of finding maximum if two numbers is
max=(a + b + (mod(a - b))) / 2;
return max;